object-api-multipart.go

```diff
@@ -243,14 +242,17 @@ func (o objectAPI) ListMultipartUploads(bucket, prefix, keyMarker, uploadIDMarke
 		}).Errorf("listMetaVolumeFiles failed with %s", e)
 		return ListMultipartsInfo{}, probe.NewError(e)
 	}
+
+	// Loop through all the received files fill in the multiparts result.
 	for _, fi := range fileInfos {
 		var objectName string
 		var uploadID string
 		if fi.Mode.IsDir() {
+			// All directory entries are common prefixes.
 			objectName = strings.TrimPrefix(fi.Name, retainSlash(bucket))
-			// For a directory entry
 			result.CommonPrefixes = append(result.CommonPrefixes, objectName)
-			continue
+			result.NextKeyMarker = objectName
```

Comment:
`result.NextKeyMarker = objectName` is being done in both if and else blocks, hence it can be after if/else blocks.